sdk-workflows/src/main/java/io/dapr/workflows/client/DaprWorkflowClient.java

```diff
@@ -14,32 +14,87 @@
 package io.dapr.workflows.client;
 
 import com.microsoft.durabletask.DurableTaskClient;
+import com.microsoft.durabletask.DurableTaskGrpcClientBuilder;
+import io.dapr.config.Properties;
+import io.dapr.utils.Version;
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
 
-public class DaprWorkflowClient {
+import javax.annotation.Nullable;
+
+public class DaprWorkflowClient implements AutoCloseable {
 
   private DurableTaskClient innerClient;
+  private ManagedChannel grpcInnerChannel;
```

Comment:
NIT: as there isn't an _outer_ channel to speak of:

```suggestion
  private ManagedChannel grpcChannel;
```